extensions/amp-analytics/amp-analytics.md

```diff
@@ -398,6 +398,33 @@ As an example, the following configuration can be used to sample 50% of the requ
 },
 ```
 
+#### Embed render start trigger (`"on": "render-start"`)
+Use this configuration to fire a request when the document or a specified embed emit `render-start` signal. This
```

Comment:
`emits`

But I'd consider switching the language to not use our internal name "signal" and just make this a plain event.